srcpkgs/libXaw3dXft/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'libXaw3dXft'
+pkgname=libXaw3dXft
+version=1.6.2h
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr --disable-static --enable-internationalization
```

Comment:
`--prefix=/usr` is specified by `xbps-src` by default, please remove.

Reply:
OK, will do.